### PR TITLE
sink: Decrease the default value of `MaxBatchSize` to avoid `Kafka Msg too large`

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
Decrease the default value of MaxBatchSize to avoid Kafka Msg too large

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: cdc/sink/codec/json.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ const (
 	// DefaultMaxMessageBytes sets the default value for max-message-bytes
 	DefaultMaxMessageBytes int = 64 * 1024 * 1024 // 64M
 	// DefaultMaxBatchSize sets the default value for max-batch-size
-	DefaultMaxBatchSize int = 4096
+	DefaultMaxBatchSize int = 16
```
Comment (amyangfei) [inline cdc/sink/codec/json.go:41]: Better to add some comments, about why `16` is used

File: cdc/sink/codec/json.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ const (
 	// DefaultMaxMessageBytes sets the default value for max-message-bytes
 	DefaultMaxMessageBytes int = 64 * 1024 * 1024 // 64M
```
Comment (overvenus) [inline cdc/sink/codec/json.go:39]: What about 1MB? 64MB exceeds Kafka's default `max.message.bytes` (1MB) https://kafka.apache.org/documentation/#brokerconfigs

Comment (leoppro) [reply cdc/sink/codec/json.go:39]: `1MB` is not a suitable default value, the max size of a value in tikv is 6MB, so the max size of a row event approximately equal to 12MB (with old value), the default value should make sure one row event at lastest can be sent to kafka

Comment (liuzix) [reply cdc/sink/codec/json.go:39]: The current implementation tries to send out the Kafka message with only one event, **no matter what**. The codec will print a warning if the message is larger than the configured value.

**Comment by liuzix** [comment]
LGTM

File: cdc/sink/codec/json.go (hunk 1 above)
Comment (leoppro) [reply cdc/sink/codec/json.go:41]: I don't know if this is a suitable value, but `4096` is too long

**Comment by amyangfei** [comment]
unit test is broken

**Pull request merged**